server/src/core/server/services/users/users.ts

```diff
@@ -1465,6 +1468,17 @@ export async function ban(
 
   let user: Readonly<User>;
 
+  const bundle = i18n.getBundle(tenant.locale);
+  const tranlsatedExplanation = translate(
```

Comment:
nit: typo `tranlsatedExplanation` -> `translatedExplanation`, not a big deal if you have bigger fish to fry